### DISCUSSION TITLE
Character length limit on text fields

### DISCUSSION THREAD

**Discussion by edthetechie** [wheels-dev/wheels#1267] [Q&A] [answered]
I'm having a really weird problem and I'm not sure if it's CFWheels or something else. We have a simple CMS system built in CFWheels which has an articles table in it. The text fields in the article are set to "ntext" in MS SQL and we're using the Froala rich text editor to let the users manage the articles. It has all been working fine but today they tried to upload a massively long article and it is being truncated. I can manually add the whole text into SQL via the SSMS console but when I then try to edit the article in the CMS the text is truncated, same thing happens on the front end when trying to pull the article text.

Does anyone know if there is some kind of limit in CFWheels on the size of a form field? I've checked our CF server settings and the max post size is set to 200Mb so I don't think that's the problem. I checked Froala and that has a character count limit plugin, but we haven't used it. I also tried changing the froala textarea to a regular textarea and the same thing happens. So this seems to be something CFWheels might be doing but I've never encountered this problem before.

If anyone has any ideas I'd be most grateful because this one has me stumped.

**Comment by neokoenig** [comment]
I don't know anything about MSSQL, but IIRC on lucee there's a "Throw error upon data truncation" option on the datasource itself with mySQL at least. If you set that to true you might get a hint as to whether it's a DB or a framework issue.

**Reply by edthetechie** [reply, marked as answer]
Thanks Tom, I don't see a setting for that in the MS SQL datasource admin but I can see a "Long Text Buffer" size which was causing the problem. I doubled it from 64000 to 128000 and the text now shows up. You get the credit for this one because you gave me the idea to check the DSN settings :-)
